api/src/main/java/org/apache/iceberg/view/ViewVersion.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.view;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A version of the view at a point in time.
+ * <p>
+ * A version consists of a view metadata file.
+ * <p>
+ * Versions are created by view operations, like Create and Replace.
+ */
+public interface ViewVersion {
+  /**
+   * Return this version's ID.
+   *
+   * @return a long ID
```

Comment:
type mismatch: int vs long. same for the parentId below.

Reply:
also is version a monotonically increased numbers? if yes, maybe document it in the Javadoc?

Reply:
Done